packages/ui/client/components/FileDetails.vue

```diff
@@ -137,7 +137,7 @@ debouncedWatch(
 )
 
 const projectNameColor = computed(() => {
-  return getProjectNameColor(current.value?.file.projectName)
+  return getProjectNameColor(current.value?.file.projectName, undefined)
```

Comment:
shouldn't we pass down the color somehow?

Reply:
Yes we should. I left this intentionally as `undefined` for now, so that I can pick up from here later. My Vue skill are very limited so it wasn't obvious where to make changes.

Reply:
I'm Vue developer now 💯